src/styles/index.css

```
@@ -523,6 +535,7 @@ html {
 html,
 body {
   background-color: var(--p-color-bg-0);
+  color: var(--p-color-default)
```

```suggestion
  color: var(--p-color-text-default)
```
I don't see a variable for `p-color-default`, should this be using the default text color?

---

Ah super good catch, ty